mule-gateway/modules/ROOT/pages/mule-gateway-proxy-config.adoc

```diff
@@ -0,0 +1,90 @@
+= Route Mule Gateway Traffic Through a Proxy Server
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can configure the Mule Gateway to route traffic through the proxy server to API Manager if your Mule Runtime engine runs inside a firewall that restricts external communication through a proxy server.
+
+Mule Gateway supports Basic, NTLM, and Kerberos v5 (via SPNEGO) authentication schemes for proxied communication.
+
+[[configure-the-proxy-server]]
+== Configure the Proxy Server
+
+To configure Mule Gateway to connect through a proxy server, you must set up your proxy server configuration in a `$MULE_HOME/conf/wrapper.conf` file. Add your proxy server information to the following properties:
```

Comment:
```suggestion
To configure Mule Gateway to connect through a proxy server, set up your proxy server configuration by adding the following properties in a `$MULE_HOME/conf/wrapper.conf` file:
```
Optional suggestion to make this shorter. Please check if it is accurate.

Reply:
Yep, it's ok